### PR TITLE
Psimao pla 119 - Adding& fixing linters

### PR DESCRIPTION
**Pull request by paulosimao-ardanlabs**

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     - misspell
+     - nolintlint
+     - prealloc
+     - staticcheck
+    # - structcheck
+    # - stylecheck
+     - typecheck
+     - unconvert
+    # - unparam
+    # - unused
```
Comment (AnthonyArdronUW) [inline .golangci.yaml:40]: unused is good thing to check in my opinion

File: business/connectors/pubsubcon/pubsubcon.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func (d *defaultConnectorImpl) GetSubMessageChannel(subid string, ret chan *pubs
 		for keepRunning {
 			// Receive blocks until the context is cancelled or an error occurs.
 			err := sub.Receive(d.ctx, func(_ context.Context, msg *pubsub.Message) {
-				logrus.Debugf("Messaged recieved: %s", msg.Data)
+				logrus.Debugf("Messaged received: %s", msg.Data)
```
Comment (AnthonyArdronUW) [inline business/connectors/pubsubcon/pubsubcon.go:51]: This log message can probably be removed or not returning the entire msg.data. However not a big issue.

File: .golangci.yaml (hunk 1 above)
Comment (paulosimao-ardanlabs) [reply .golangci.yaml:40]: We will evolve with more linters. This is just a starting point.

File: business/connectors/pubsubcon/pubsubcon.go (hunk 2 above)
Comment (paulosimao-ardanlabs) [reply business/connectors/pubsubcon/pubsubcon.go:51]: oooops - sure

**Review by AnthonyArdronUW — APPROVED** [review]

**Pull request merged**